api/user.go

```diff
@@ -1059,10 +1059,11 @@ func getProfiles(c *Context, w http.ResponseWriter, r *http.Request) {
 			if c.IsSystemAdmin() {
 				options["fullname"] = true
 				options["email"] = true
+			} else {
+				p.ClearNonProfileFields()
```

Comment:
Does the system admin need these fields?  also what about getuser() webservice call?  should we return the private mfa code stuff?